Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml

```diff
@@ -3,6 +3,7 @@
   id: RMCAdminObserver
   noSpawn: true
   components:
+  - type: Marine
```

Comment:
smug said to remove this in the end for potential issues, we'll have a proper component for the purpose of this later
```suggestion
```